insights/parsers/corosync_cmapctl.py

```diff
@@ -0,0 +1,65 @@
+"""
+CorosyncCmapctl - Command ``corosync-cmapctl [params]``
+=======================================================
+
+This module parses the output of the ``corosync-cmapctl [params]`` command.
+"""
+
+from insights import parser, CommandParser, LegacyItemAccess
+from insights.parsers import SkipException, ParseException
+from insights.specs import Specs
+
+
+@parser(Specs.corosync_cmapctl)
+class CorosyncCmapctl(CommandParser, LegacyItemAccess):
```

Comment:
Please inherit it from `dict` instead of `LegacyItemAccess`

Reply:
> Please inherit it from `dict` instead of `LegacyItemAccess`

Sure, I'll update it, thanks.